pom.xml

```diff
@@ -175,6 +175,7 @@ under the License.
 		<orc.version>1.5.6</orc.version>
 		<japicmp.referenceVersion>1.18.0</japicmp.referenceVersion>
 		<japicmp.outputDir>tools/japicmp-output</japicmp.outputDir>
+		<spotless.skip>false</spotless.skip>
```

Comment:
I still don't fully understand the need here. Skipping the spotless plugin seems to work with the existing system properties {{spotless.check.skip}} or/and {{spotless.apply.skip}}. :thinking:

Reply:
> Skipping the spotless plugin seems to work with the existing system properties spotless.check.skip or/and spotless.apply.skip.

huh, true, but we hardly use those anywhere. We have 3 references to `spotless.skip`, but only 1 to `spotless.check.skip`.

Reply:
we could of course update them to use `spotless.check.skip`, but at least I `expected` `spotless.skip` to be the property to use.